kubectl-rook-ceph.sh

```diff
@@ -27,24 +27,26 @@ function print_usage() {
+  echo "  -o, --operator-namespace='rook-ceph'                        : the namespace of the rook operator"
+  echo " --context=<context_name>                                     : the name of the Kubernetes context to be used"
 
   echo "COMMANDS"
-  echo "  ceph <args>                               : call a 'ceph' CLI command with arbitrary args"
-  echo "  rbd <args>                                : call a 'rbd' CLI command with arbitrary args"
+  echo "  ceph <args>                                                 : call a 'ceph' CLI command with arbitrary args"
+  echo "  rbd <args>                                                  : call a 'rbd' CLI command with arbitrary args"
   echo "  operator <subcommand>..."
-  echo "    restart                                 : restart the Rook-Ceph operator"
-  echo "    set <property> <value>                  : Set the property in the rook-ceph-operator-config configmap."
-  echo "  mons                                      : output mon endpoints"
+  echo "    restart                                                   : restart the Rook-Ceph operator"
+  echo "    set <property> <value>                                    : Set the property in the rook-ceph-operator-config configmap."
+  echo "  mons                                                        : output mon endpoints"
   echo "  rook <subcommand>..."
-  echo "    version                                 : print the version of Rook"
-  echo "    status                                  : print the phase and conditions of the CephCluster CR"
-  echo "    status all                              : print the phase and conditions of all CRs"
-  echo "    status <CR>                             : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
-  echo "    purge-osd <osd-id> [--force]            : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
+  echo "    version                                                   : print the version of Rook"
+  echo "    status                                                    : print the phase and conditions of the CephCluster CR"
+  echo "    status all                                                : print the phase and conditions of all CRs"
+  echo "    status <CR>                                               : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
+  echo "    purge-osd <osd-id> [--force]                              : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
+  echo " create-debug [--name deployment] [--image <alternate-image>] : Make the OSD or mon in debug mode"
+  echo " remove-debug [--name deployment]                             : Make the OSD or mon out of debug mode"
```

Comment:
@travisn 
Here it should be the deployment name[rook-ceph-mon-b] or the debug deployment name[rook-ceph-mon-b-debug].

Currently using the  `deployment name[rook-ceph-mon-b]`

Reply:
I would say it should accept either.

Reply:
That means it should accept `rook-ceph-mon-b` and `rook-ceph-mon-b-debug` both?
@BlaineEXE

Reply:
That was my thought. Bash has built-in basic string manipulation. You can remove a suffix from a string pretty easily.

```sh
base_name="${input_deployment_name%'-debug'}" # strip '-debug' suffix
# the base_name is always rook-ceph-mon-a regardless of whether
# input_deployment_name has the -debug suffix or not
```

Reply:
yaa thankx